src/pages/AddPersonalBankAccountPage.js

```diff
@@ -89,15 +96,13 @@ class AddPersonalBankAccountPage extends React.Component {
      * @param {Object} values.password The password of the user adding the bank account, for security.
      */
     submit(values) {
-        const selectedPlaidBankAccount = _.findWhere(lodashGet(this.props.plaidData, 'bankAccounts', []), {
-            plaidAccountID: this.state.selectedPlaidAccountID,
-        });
-
+        const selectedPlaidBankAccount = this.getSelectedPlaidAccount();
         BankAccounts.addPersonalBankAccount(selectedPlaidBankAccount, values.password);
     }
 
     render() {
         const shouldShowSuccess = lodashGet(this.props, 'personalBankAccount.shouldShowSuccess', false);
+        const isPlaidBankAccountSelected = Boolean(this.state.selectedPlaidAccountID) && !_.isEmpty(this.getSelectedPlaidAccount());
```

Comment:
You can probably get rid of `Boolean(this.state.selectedPlaidAccountID) &&` ?

Reply:
Agree, `!_.isEmpty(this.getSelectedPlaidAccount())` will do the job.

Reply:
Yes, but I think it will be better to short circuit and not call the function at all